proposals/lip-0053.md

```diff
@@ -133,28 +144,247 @@ The following constants are used throughout the document:
 | Name                                                     | Type   | Value                   |
 |----------------------------------------------------------|--------|-------------------------|
 | **Interoperability Constants**                           |        |                         |
-| `MODULE_ID_INTEROPERABILITY`                             | uint32 | 64                      |
+| `MODULE_ID_INTEROPERABILITY`                             | bytes  | TBD                     |
 | `STORE_PREFIX_CHAIN_DATA`                                | bytes  | 0x8000                  |
-| `LIVENESS_LIMIT`                                         | uint32 | `30*24*3600`            |
+| `LIVENESS_LIMIT`                                         | uint32 | `30*24*60*60`           |
 | **Interoperability Command and Cross-chain Command IDs** |        |                         |
-| `COMMAND_ID_SIDECHAIN_CCU`                               | uint32 | 2                       |
-| `COMMAND_ID_MAINCHAIN_CCU`                               | uint32 | 3                       |
+| `COMMAND_ID_SIDECHAIN_CCU`                               | bytes  | 0x0002                  |
+| `COMMAND_ID_MAINCHAIN_CCU`                               | bytes  | 0x0003                  |
 | **Chain Status**                                         |        |                         |
-| `CHAIN_REGISTERED`                                       | uint32 | 0                       |
-| `CHAIN_ACTIVE`                                           | uint32 | 1                       |  
-| `CHAIN_TERMINATED`                                       | uint32 | 2                       |
+| `CHAIN_STATUS_REGISTERED`                                | uint32 | 0                       |
+| `CHAIN_STATUS_ACTIVE`                                    | uint32 | 1                       |  
+| `CHAIN_STATUS_TERMINATED`                                | uint32 | 2                       |
 | **Message Tags**                                         |        |                         |
 | `MESSAGE_TAG_CERTIFICATE `                               | bytes  | "LSK_CE_" ASCII-encoded |
 
 Several of those constants are shared with the other LIPs defining the Interoperability module and all of the needed constants for the Interoperability module are defined in [LIP 0045][lip-0045]. That LIP should be considered correct if a value stated here differs.
```

Comment:
If we acknowledge that we may not be able to keep multiple constant tables in sync and the above version may be incorrect, then I would refer to the interoperability module for the constants instead of having a redundant definition here. This is also lower maintenance for us. 

For the message tag, it would also be nice to reference LIP 61 where it is original defined.